features/dynamic-bundle-discounts.md

```diff
@@ -659,4 +643,53 @@ if nostoBundleHandler.isValid()
 end
 
 Output.cart = Input.cart
-```
\ No newline at end of file
+```
+
+## Troubleshooting
+
+### Issue #1: When additional content displayed in cart page
+
+![Properties Exposed](https://user-images.githubusercontent.com/82023195/168611801-666c24bc-c7f9-4edd-bdf5-2be32798391c.png)
+
+### Cause
+
+Nosto implementation uses a private property field for sharing discount information with Shopify. In newer theme versions, Shopify automatically hides these fields while displaying products in cart page. In older version of themes, these fields gets exposed as shown below.
+
+### Solution
+
+In such a case, make the following changes to the template liquid file that displays the cart information (usually cart.liquid or cart-template.liquid).
+
+```html
+{% raw %}
+{% for p in item.properties %}
+  {% assign first_character_in_key = p.first | truncate: 1, '' %} <== new line
+  {% unless p.last == blank or first_character_in_key == '_' %} <== new line
+  <small>
+    {{ p.first }}:
+
+    {% comment %}
+      Check if there was an uploaded file associated
+    {% endcomment %}
+    {% if p.last contains '/uploads/' %}
+      <a href="{{ p.last }}">{{ p.last | split: '/' | last }}</a>
+    {% else %}
+      {{ p.last }}
+    {% endif %}
+  </small><br />
+  {% endunless %} <== new line
+{% endfor %}
+{% endraw %}
+```
+
+### Issue #2: When discount is not applied
+
+### Cause
+
+1. Nosto provided scripts are not used as-is. Nosto can't provide support in this case.
+2. Incorrect value used for SECRET_KEY. 
```

Comment:
```suggestion
2. The `SECRET_KEY` in the line item script is incorrect.
```

Reply:
I think the new sentence is little confusing. It suggests that the `SECRET_KEY` is incorrect rather than it's value